src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/transformer/cas1/Cas1OutOfServiceBedTransformerTest.kt

```diff
@@ -69,8 +99,26 @@ class Cas1OutOfServiceBedTransformerTest {
     verify { cas1OutOfServiceBedCancellationTransformer.transformJpaToApi(any()) wasNot Called }
   }
 
-  @Test
-  fun `transformJpaToApi transforms correctly when cancelled`() {
+  @CsvSource(
+    value = [
+      "-10,-10,1,past",
+      "-10,-5,6,past",
+      "-5,5,11,current",
+      "-1,0,2,current",
+      "0,0,1,current",
+      "0,1,2,current",
+      "5,10,6,future",
+    ],
+  )
+  @ParameterizedTest
+  fun `transformJpaToApi transforms correctly when cancelled`(
```

Comment:
could we slim this spec right down maybe? Does it perhaps only need to assert:

```
assertThat(result.status).isEqualTo(Cas1OutOfServiceBedStatus.cancelled)
assertThat(result.cancellation).isEqualTo(cancellation)
```

if so, then maybe making it much smaller will help readers understand the change in behaviour which is being described?

Reply:
I've updated this.